.gitignore

```diff
@@ -91,6 +91,8 @@ playground.xcworkspace
 
 Pods/
 .Podfile.swp
+.Podfile.lock.swp
```

Comment:
last but not least, try to don´t open these files on Vi or ignore the extension if you really need that, it will create more of these and they´re not ios related, only Vi related